deployment/dev/aks-periscope.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: v1
```

Comment:
💡 We really need to talk about this please? We are duplicating deployment files and this is kind of a design thought, I need to think more. And not making this part of a refactor. This should be more broader discussed before we go ahead and duplicate deployment files for small use-cases.